build/webpack/webpack.extension.node.config.js

```diff
@@ -171,12 +191,24 @@ const config = {
                 { from: './node_modules/@aminya/node-gyp-build/**/*' },
                 {
                     from: './node_modules/zeromq/**/*',
-                    filter: shouldCopyFileFromZmqFolder.bind(this, './node_modules/zeromq')
-                },
+                    filter: (filepath) =>
+                        shouldCopyFileFromZmqFolder(
+                            path.join(constants.ExtensionRootDir, 'node_modules', 'zeromq'),
+                            filepath
+                        )
+                }
+            ]
+        }),
+        new copyWebpackPlugin({
+            patterns: [
                 // Copy files from fallback zmq package.
                 {
                     from: './node_modules/zeromqold/**/*',
-                    filter: shouldCopyFileFromZmqFolder.bind(this, './node_modules/zeromqold')
```

Comment:
This was the root cause, on windows the path sep was `\`
However modified all of the code to be platform agnostic for the builds.